### DISCUSSION TITLE
Install for MacBook Pro 2019 Intel

### DISCUSSION THREAD

**Discussion by tomfelder94** [KhronosGroup/KTX-Software#723] [Q&A] [answered]
Hi guys, I have not been able to install the KTX-Software successfully on my mac. Which binary do I need exactly and how can I install it then so that I can use it globally from the terminal?

Thanks for your help!

**Comment by MarkCallow** [comment, marked as answer]
What binary were you trying, how did you attempt to install it and what happened?

For Intel Macs you need a `Darwin-x86-64` package. You can find the latest release at https://github.com/KhronosGroup/KTX-Software/releases/download/v4.2.1/KTX-Software-4.2.1-Darwin-x86_64.pkg.

After downloading the package, open it and the installer will start. The package and contents are signed and notarized so it installs without any issues. If you want to install the optional Development (header files) or Java (libktx_jni) components click the Customize button which will appear during the installation process after you agree to the license. By default it installs the tools in `/usr/local/bin` so they will already be in $PATH for any user.

**Comment by tomfelder94** [comment]
Thanks a lot, this worked for me.

From: Mark Callow ***@***.***>
Date: Friday, 30 June 2023 at 11:50
To: KhronosGroup/KTX-Software ***@***.***>
Cc: tomfelder94 ***@***.***>, Author ***@***.***>
Subject: Re: [KhronosGroup/KTX-Software] Install for MacBook Pro 2019 Intel (Discussion #723)

What binary were you trying, how did you attempt to install it and what happened?

For Intel Macs you need a Darwin-x86-64 package. You can find the latest release at https://github.com/KhronosGroup/KTX-Software/releases/download/v4.2.1/KTX-Software-4.2.1-Darwin-x86_64.pkg.

After downloading the package, open it and the installer will start. The package and contents are signed and notarized so it installs without any issues. If you want to install the optional Development (header files) or Java (libktx_jni) components click the Customize button which will appear during the installation process after you agree to the license. By default it installs the tools in /usr/local/bin so they will already be in $PATH for any user.
